optional/fetchmail/fetchmail.py

```diff
@@ -66,10 +67,9 @@ def run(debug):
         for fetch in fetches:
             fetchmailrc = ""
             options = "options antispam 501, 504, 550, 553, 554"
-            if "FETCHMAIL_OPTIONS" in os.environ: options += f'{ os.environ["FETCHMAIL_OPTIONS"]}'
+            if "FETCHMAIL_POLL_OPTIONS" in os.environ: options += f'{ os.environ["FETCHMAIL_POLL_OPTIONS"]}'
```

Comment:
```suggestion
            if "FETCHMAIL_POLL_OPTIONS" in os.environ: options += f' {os.environ["FETCHMAIL_POLL_OPTIONS"]}'
```
Space is in wrong position.